### DISCUSSION TITLE
Layouts for blocks

### DISCUSSION THREAD

**Discussion by andrewpbray** [quarto-dev/quarto-cli#1508] [Q&A]
While reading through the docs at [Guide >> Authoring >> Figures](https://quarto.org/docs/authoring/figures.html), I came across the different layout attributes that can be specified. At the end of the section, [it mentions](https://quarto.org/docs/authoring/figures.html#block-layout) that these layouts work for any block element - that's great.

That made me turn to the reference section to find the list of all of the different layouts and their options / specs. The only one I could find was in [Code Cells > knitr > Panel Layout](https://quarto.org/docs/reference/cells/cells-knitr.html#panel-layout). In that section there is mention of `layout-align`. I tried using this on a block element but it didn't quite work.

```
---
format: pdf
---

:::{layout-ncol=2 layout-align=left}

- 1
- 2
- 3

this is left aligned
:::

:::{layout-ncol=2 layout-align=center}

- 1
- 2
- 3

this is centered
:::


:::{layout-ncol=2 layout-align=right}

- 1
- 2
- 3

this is right aligned
:::
```

<img width="502" alt="Screen Shot 2022-07-23 at 9 23 50 AM" src="https://user-images.githubusercontent.com/1627362/180613851-87c002e5-67be-4f7e-bcdf-4bd2b7b1ef5b.png">

Is this is a bug/oversight or is `layout-align` just meant to be a cell attribute? Would it make sense to make a separate reference section for the layouts that work on block elements?

**Comment by mcanouil** [comment]
If I am not mistaken those layout options are for knitr cell outputs not div blocks as in your example.

Multiple columns are described in the following page (and some other places, you can search in the website for "multiple columns"): https://quarto.org/docs/presentations/index.html#multiple-columns
